core/src/scrim/scrim.service.ts

```diff
@@ -155,9 +157,22 @@ export class ScrimService {
                         break;
                     case EventTopic.ScrimCreated:
                         this.pubsub.publish(this.pendingScrimsSubTopic, {followPendingScrims: v.payload}).catch(this.logger.error.bind(this.logger));
+                        this.pubsub.publish(this.allActiveScrimsSubTopic, {
```

Comment:
It might make more sense to put this before / after the switch, just checking if it isn't a scrim metrics update.

Reply:
Done.